internal/envoy/bootstrap.go

```diff
@@ -102,6 +105,20 @@ func (c *BootstrapConfig) GetAdminAccessLogPath() string {
 func (c *BootstrapConfig) GetDNSLookupFamily() string {
 	return stringOrDefault(c.DNSLookupFamily, "auto")
 }
+
+// ValidAdminAddress checks if the address supplied is
+// "localhost" or an IP address. Only a Unix Socket
+// is supported for this address to mitigate security.
+func ValidAdminAddress(address string) error {
+	// Value of "localhost" is invalid.
+	if address == "localhost" ||
+		net.ParseIP(address) != nil ||
+		!strings.Contains(address, "/") {
```

Comment:
edge case, but someone could conceivably want to use a relative file path with just the file name here (e.g. just `foo.sock`), which could be valid

maybe we should make the validation be an absolute file path to prevent this and make sure people are exact (e.g. check for `/` at the start of the string)? or relax the validation a bit to not require a slash?

could also make it check for https://pkg.go.dev/os#PathSeparator so its not another step to cross-platform convert if we want to in the future

Reply:
Yeah you'd have to map the path of that to where Envoy is running to make that work. Maybe this should change to just the path and not the name? Just trying to keep this super simple for folks. 

I'll take it out and not require the `/`.